sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -161,6 +161,30 @@ export class BatchingReceiver extends MessageReceiver {
         }
       };
 
+      // Use new message wait timer only in peekLock mode
+      if (this.receiveMode === ReceiveMode.peekLock) {
+        /**
+         * Resets the timer when a new message is received. If no messages were received for
+         * `newMessageWaitTimeoutInSeconds`, the messages received till now are returned. The
+         * receiver link stays open for the next receive call, but doesnt receive messages until then
+         */
+        this.resetTimerOnNewMessageReceived = () => {
```

Comment:
What is the reason for moving this code from its previous location?

Reply:
It needed finalAction() to be defined and the IDE/compiler was showing an error

Reply:
That's weird... I tried moving it back to its original location on my machine and there are no errors being shown in the editor or by `rushx build`

Reply:
Yes, it isn't showing now - moved it back